### PR TITLE
Arrowhead

### PR DESCRIPTION
**Pull request by theundeadmonk**
Refactored code as per review comments

### CODE REVIEW

File: arrowhead/arrowhead.rb [hunk 1]
```diff
@@ -1,34 +1,47 @@
 class Arrowhead
-  # This seriously belongs in a database.
+
+  UNKNOWN_METHOD = "Unknown region, please provide a valid region."
+  UNKNOWN_SHAPE = %"Unknown shape value. Are you sure you know what you're
+    talking about?"
+# This seriously belongs in a database.
```
Comment (ashtonthomas) [inline arrowhead/arrowhead.rb]: Hey, @theundeadmonk what does the `%` do before the string?

File: arrowhead/arrowhead.rb [hunk 2]
```diff
@@ -1,34 +1,47 @@
...
-      shapes = CLASSIFICATIONS[region]
-      if shapes.include? shape
-        arrowhead = shapes[shape]
-        puts "You have a(n) '#{arrowhead}' arrowhead. Probably priceless."
-      else
-        raise "Unknown shape value. Are you sure you know what you're talking about?"
-      end
-    else
-      raise "Unknown region, please provide a valid region."
+    arrowhead = %"You have a(n) '#{CLASSIFICATIONS[region][shape]}'
```
Comment (ashtonthomas) [inline arrowhead/arrowhead.rb]: Do you think it is better to have the query after the shape and region checks?

**Comment by ashtonthomas** [comment]
@theundeadmonk, It would be cool if all of this conformed to hound/rubocop, but I don't think we need to do another review. I made a couple of comments/questions. Address as you see fit.

LGTM

File: arrowhead/arrowhead.rb (hunk 1 above)
Comment (theundeadmonk) [inline arrowhead/arrowhead.rb]: So the % lets me span the string across multiple lines. That way, line length won't exceed 80 characters.

File: arrowhead/arrowhead.rb (hunk 2 above)
Comment (theundeadmonk) [inline arrowhead/arrowhead.rb]: Yeah, I've made the changes.

**Pull request closed without merging**